theme/models.py

```diff
@@ -167,7 +167,7 @@ class QuotaMessage(models.Model):
                                                             'or to acquire additional quota, after '
                                                             'which you will no longer be able to '
                                                             'create new resources in HydroShare. ')
-    enforce_content_prepend = models.TextField(default='Your action to add content to HydroShare '
+    enforce_content_prepend = models.TextField(default='Your action '
```

Comment:
@pkdash Agreed it'd be better to move this default out of direct model field definition. After doing a bit of research, it takes too much hassle to do this at this point since I don't want to squash all migrations from this app especially considering there is a data migration involved. I think the best to do this at this point is to directly modify the migration file to change this default text so that no new migration file is generated. In fact, this default does not have to be changed, the text can be modified by admin from admin panel anyway, but I want a good default to go in before quota goes live. After that, I don't think these default messages need to be changed any more.